rabbitmq-producer/src/main/java/com/example/micrometer/RabbitProducerApplication.java

```diff
@@ -0,0 +1,83 @@
+package com.example.micrometer;
+
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import io.micrometer.tracing.Tracer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.Exchange;
+import org.springframework.amqp.core.FanoutExchange;
+import org.springframework.amqp.core.MessageListener;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.WebApplicationType;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Service;
+
+@SpringBootApplication
+public class RabbitProducerApplication implements CommandLineRunner {
+
+    private static final String EXCHANGE_NAME = "fanout.ex";
+
+    private static final String QUEUE_NAME = "queue.ex";
```

Comment:
What is `ex`? Is it short for exchange? If so, I might call it `test.queue` or something because it is not an Exchange. I think the same thing applies to `RabbitConsumerApplication`.